Orange/canvas/application/canvasmain.py

```diff
@@ -2221,6 +2226,17 @@ def tr(self, sourceText, disambiguation=None, n=-1):
 
     def __update_from_settings(self):
         settings = QSettings()
+
+        settings.beginGroup("tick_rate")
+        tick_rate = settings.value('tick_rate_value', defaultValue=FrequencyUIDefaultValue).value
+        settings.endGroup()
+        try:
+            value = int(float(tick_rate))
+        except:
+            warnings.warn('Tick rate value '+tick_rate+' is not allowed. Using default instead.')
+            value = int(FrequencyUIDefaultValue)
+        self.scheme_widget.scheme().signal_manager.frequency = value
```

Comment:
I think that the signal_manager needs to use the frequency as per settings also when it's being constructed, unless this func is guaranteed to be called in all cases where the signal manager is being recreated.